### DISCUSSION TITLE
Is the "Create Item" Function broken?

### DISCUSSION THREAD

**Discussion by AngryCrafter** [AngryBeaver/beavers-crafting#135] [Q&A]
Ive done your troubleshooting instructions, it was hard for me to understand them and still cant seem to get it after hours. I really need help :(
![318173716-0d70a282-1279-443d-bba0-1bcc1caa0300](https://github.com/AngryBeaver/beavers-crafting/assets/165454065/f4638241-e46b-4f71-906f-e2e42a2cb5cf)

**Comment by AngryBeaver** [comment]
https://github.com/AngryBeaver/beavers-crafting/issues/134
